README.md

```diff
@@ -30,7 +30,7 @@ So far, this feels like how the book is gonna play out:
   - [ ] HTTP/1, HTTP/2 and HTTP/3
     - [ ] HTTP/2 is a real world thing today use it
     - [ ] Lets Stop Building Around a Network Hack
-  - [ ] Asynchronous Methodologies
+  - [x] Asynchronous Methodologies
```

Comment:
Did you think about including HTTP Streaming APIs in this chapter?

Reply:
Yessir! I'll get there.